_episodes/12-cluster.md

```diff
@@ -23,143 +27,278 @@ On the first day, her supervisor greets her friendly and welcomes her to the job
 
 Lola walks to the IT department and finishes the paper work to get an account. One of the admins promises to sit down with her in the morning to show her the way around the machine. The admin explains that Lola will use a small to mid-range HPC cluster.
 
-{% include links.md %}
+## Going remote
+
+First of all, the admin asks Lola to connect to the super computer. She mentions that in the past, compute clusters were named after planets or moons as they often presented distant somewhat mythological places. One of the admin's first instructors then often said, that they would use the Space Shuttle (or `ssh` briefly) to reach that planet or moon. So the admin asks Lola to open a terminal on her laptop and type in the following commands:
+
+~~~ 
+$ ssh lola@{{ site.workshop_login_host }}
+~~~
+{: .language-bash}
+
+> ## Logging in
+> 
+> If you do this material on your own, be sure to replace `lola` with the username that is attributed to you on {{ site.workshop_login_host }}. When you hit enter, a prompt like this might appear:
+>
+> ~~~
+> lola@{{ site.workshop_login_host }}'s password:
+> ~~~
+> {: .output}
+> 
+> Now is your chance to type in your password. But watch out, the characters you type are not displayed on the screen.
+{: .callout}
+
+~~~ 
+Last login: Fri Dec 14 14:13:14 2018 from lolas_laptop
+-bash-4.1$ 
+~~~
+{: .output}
 
-## Where are we?
+The admin explains to Lola that she is using a program known as the secure shell or `ssh`. This establishes a temporary encrypted connection between Lola's laptop and `{{ site.workshop_login_host }}`. The word before the `@` symbol, e.g. `lola` here, is the user account name that Lola has access permissions for on the cluster. 
 
-Go ahead and log in to the cluster.
-```
-[user@laptop]$ ssh remote
-```
-{: .bash}
+> ## Where do I get this `ssh` from ?
+> On Linux and/or macOS, the `ssh` command line utility is almost always pre-installed. Open a terminal and type `ssh --help` to check if that is the case. 
+> 
+> At the time of writing, the openssh support on Microsoft is still very [recent](https://blogs.msdn.microsoft.com/powershell/2017/12/15/using-the-openssh-beta-in-windows-10-fall-creators-update-and-windows-server-1709/). Alternatives to this are [putty](http://www.putty.org), [bitvise SSH](https://www.bitvise.com/ssh-client-download) or [mRemoteNG](https://mremoteng.org/). Download it, install it and open the GUI. The GUI asks for your user name and the destination address or IP of the computer you want to connect to. Once provided, you will be queried for your password just like in the example above.
+{: .callout}
```

Comment:
I've seen a number of sucessfull user of https://mobaxterm.mobatek.net/ which also offer file transfer in a GUI.

Reply:
this section needs some some care in the future. Maybe even land in `Extras -> Setup`. added with a4a14a2